lighthouse-core/gather/drivers/driver.js

```diff
@@ -210,6 +211,42 @@ class Driver {
   }
 
   /**
+   * Checks all serviceworkes and see if any duplications are running
+   * @param {string} pageUrl
+   */
+  checkForMultipleServiceWorkers(pageUrl) {
+    // Get necessary information of serviceWorkers
+    const getRegistrations = new Promise(resolve => {
```

Comment:
maybe pull this out into a `getServiceWorkerRegistrations`, like `getServiceWorkerVersions`? That way `ServiceWorker.enable`/`ServiceWorker.disable` is also handled